railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_7_0.rb.tt

```diff
@@ -6,6 +6,14 @@
 #
 # Read the Guide for Upgrading Ruby on Rails for more info on each option.
 
+# Change the return value of `ActionDispatch::Request#content_type` to the
+# Content-Type header without modification.
+# Rails.application.config.action_dispatch.return_only_request_media_type_on_content_type = false
```

Comment:
This should not be here since we want users to see the deprecation, so they should not be flipping this config

Reply:
I've removed it. I also added the other two to the configuration guide.